stage1/init/init.go

```diff
@@ -135,7 +136,7 @@ func init() {
 	flag.Var(&privNet, "private-net", "Setup private network")
 	flag.BoolVar(&interactive, "interactive", false, "The pod is interactive")
 	flag.StringVar(&mdsToken, "mds-token", "", "MDS auth token")
-
+	flag.StringVar(&localConfig, "local-config", common.DefaultLocalConfigDir, "Local config path")
```

Comment:
We need to document this in the [stage1 implementors guide](https://github.com/coreos/rkt/blob/master/Documentation/devel/stage1-implementors-guide.md).